bobtemplates/django/+package.fullname+/README.md.bob

```diff
@@ -0,0 +1,124 @@
+# {{{package.fullname}}}
+
+## Development
+
+### Setup
+
+A few steps are needed to setup the backend.
+Run these steps within the project folder.
+
+1. Create and activate a python virtual environment:
+    `python3 -m venv . && source bin/activate`
+
+2. Install the application and its requirements:
+    `pip install -e .[development]`
```

Comment:
and `.[development]` should also be put in quotes in order to not let shells (zsh) try to evaluate it.

Reply:
`bin/setup`, then we won't need to put the braces in quotes.